totest-manager.py

```diff
@@ -399,12 +399,18 @@ def package_ok(self, project, package, repository, arch):
         url = self.api.makeurl(['build', project, '_result'], query)
         f = self.api.retried_GET(url)
         root = ET.parse(f).getroot()
-        for repo in root.findall('result'):
-            status = repo.find('status')
-            if status.get('code') != 'succeeded':
-                logger.info(
-                    '%s %s %s %s -> %s' % (project, package, repository, arch, status.get('code')))
-                return False
+        succeeded = root.findall('result/status[@code="succeeded"]')
+        # [@code!='succeeded'] is not supported by ET
```

Comment:
If you wanted the `!=` support you can use `lxml` as done all over the code now.

https://github.com/openSUSE/openSUSE-release-tools/blob/33a4a3b3484e536062a1e75d0398a99f8c8b7c91/osclib/core.py#L7

Otherwise, fine by me.

Reply:
For now this should be enough - this is supposed to be a simple fix only.

Reply:
This makes it simpler, not more complex, but feel free to merge.

Reply:
> feel free to merge.

I don't have the permissions to do that myself.

Reply:
You have now